pkg/build/types/parser/deffile.go

```diff
@@ -169,7 +219,7 @@ func doSections(s *bufio.Scanner, d *types.Definition) error {
 			}
 		} else {
 			//this is a section
```

Comment:
I would add a space here, for consistency:

```suggestion
 			// this is a section
```